src/main/java/com/onarandombox/MultiverseCore/listeners/MVPlayerListener.java

```diff
@@ -304,9 +304,6 @@ public void playerPortal(PlayerPortalEvent event) {
                     + "' was allowed to go to '" + event.getTo().getWorld().getName()
                     + "' because enforceaccess is off.");
         }
-        if (!plugin.getMVConfig().isUsingDefaultPortalSearch() && event.getPortalTravelAgent() != null) {
-            event.getPortalTravelAgent().setSearchRadius(plugin.getMVConfig().getPortalSearchRadius());
```

Comment:
So is there no replacement for the Travel Agent? Just more curious than anything. If it truly is gone, I think @dumptruckman should accept this at his leisure, but if there's a replacement, that should be implemented instead.

Reply:
The replacement is to use the portal events.